src/styles/main.scss

```
@@ -1,3 +1,65 @@
+$calendar-width: 707px;
+
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  width: $calendar-width;
+  margin: 100px auto;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @for $i from 1 through 31 {
+    div.calendar__day:nth-child(#{$i})::before {
+      content: "#{$i}";
+    }
+  }
+
+  &__day {
+    height: 100px;
+    width: 100px;
+    border: 1px solid black;
+    background-color: #eee;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 30px;
+    transition-duration: 0.5s;
+    transition-timing-function: ease;
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+$calendar-start-day:
+  "mon" 0,
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
```

Please don't use hard-coded values. Change it to indexes and you will be able to calculate margins basing on that
```suggestion
$calendar-start-day:
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
etc...
```